packages/ci-v5/package copy.json

```diff
@@ -0,0 +1,55 @@
+{
```

Comment:
Is this file used? 
If so, can you replace the space in the file name with an underscore?
"package_copy.json"

Reply:
Ew, how did this get in here? Must have happened during a rebase. Removing. Thanks for catching it.